src/test/regress/output/multi_copy.source

```diff
@@ -640,8 +640,8 @@ SELECT master_create_worker_shards('super_packed_numbers_hash', 4, 1);
  
 (1 row)
 
-COPY (SELECT 1, ROW(ROW(42, 42), ROW(42, 42))) TO '/tmp/copy_test_composite_of_composite';
-COPY super_packed_numbers_hash FROM '/tmp/copy_test_composite_of_composite';
+COPY (SELECT 1, ROW(ROW(42, 42), ROW(42, 42))) TO '@testtablespace@\copy_test_composite_of_composite';
```

Comment:
Why are we using `@testtablespace@` as the temp directory? Can't we use something like `mktemp` which seems to be cross-platform?